gui/mainwindow.cc

```diff
@@ -869,7 +895,7 @@ bool MainWindow::runGpsbabel(const QStringList& args, QString& errorString,
 {
   QProcess* proc = new QProcess(nullptr);
   QString name = "gpsbabel";
-  proc->start(name, args);
+  proc->start(QApplication::applicationDirPath() + '/' + name, args);
```

Comment:
Consider refactoring this and the one above in 505 to use a simple one-liner that builds the full path to the executable in a single place before commit.